extensions/spark/kyuubi-spark-authz/src/main/resources/database_command_spec.json

```diff
@@ -215,4 +215,4 @@
   } ],
   "opType" : "SWITCHDATABASE",
   "uriDescs" : [ ]
-} ]
+} ]
```

Comment:
I don't know why this pr needs to change this...But without this change, GA failed.

Reply:
cc @yaooqinn